documentation/product/settings/topics/ai-assistant.md

```diff
@@ -0,0 +1,13 @@
+# AI Assistant
+
+Administrators can enable the AI Chat Assistant. This feature is designed to help users answer questions about the application, perform safe actions such as navigation and summarization.
```

Comment:
"Administrators can enable the AI Assistant." -- we should be consistent with the feature name 

"questions about the application, and perform safe actions"

Reply:
Done, Thanks!